libmscore/rendermidi.cpp

```diff
@@ -731,6 +735,478 @@ bool Score::isSubdivided(ChordRest* chord, int swingUnit)
             return false;
       }
 
+void renderTremolo(Chord *chord, QList<NoteEventList> & ell){
```

Comment:
Hi Nicolas, yes these functions don’t have comments.

The functions are their because I took one huge function that spanned many screen-fulls, and divided them into
individual functions the names of which describe what the code does.  
The code is more or less the same as before.  The code had no comments before I touched it, and still does not.  

I can try to invent a comment for the functions, but in my opinion, if the deserve a comment now, then they
already deserved a comment before I touched the code.

What do you think?


> On 18 May 2015, at 12:36, Nicolas Froment <notifications@github.com> wrote:
> 
> In libmscore/rendermidi.cpp <https://github.com/musescore/MuseScore/pull/1946#discussion_r30496906>:
> 
> > @@ -731,6 +735,478 @@ bool Score::isSubdivided(ChordRest* chord, int swingUnit)
> >              return false;
> >        }
> >  
> > +void renderTremolo(Chord *chord, QList<NoteEventList> & ell){
> Indentation of these new functions is completely off. There is no comment before the function. Idem for renderArpeggio()
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/musescore/MuseScore/pull/1946/files#r30496906>.
>